apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/BigtableClientTest.cs

```diff
@@ -28,6 +28,113 @@ public class BigtableClientTest
         private class TestBigtableClient : BigtableClient { }
 
         [Fact]
+        public async Task CheckAndMutateRow_Validate_TableName()
+        {
+            var client = new TestBigtableClient();
+            await CheckAndMutateRow_ValidateArguments<ArgumentNullException>(
+                null,
+                "abc",
+                RowFilters.PassAllFilter(),
+                new[] { Mutations.DeleteFromRow() },
+                new[] { Mutations.DeleteFromRow() });
+        }
+
+        [Fact]
```

Comment:
Could use a theory with a property returning the different invalid row keys?

See https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/master/apis/Google.Cloud.Firestore.Data/Google.Cloud.Firestore.Data.Tests/WriteBatchTest.SentinelValues.cs for some simple examples.

(Applies to various tests here.)

Reply:
Right now I'm not doing any validation of row keys other than checking for non-empty. The RPC docs don't mention any constraints on the row key for these requests, so I don't want to bake in any extra validation just incase the implementation changes.